sw/device/tests/sim_dv/lc_ctrl_transition_test.c

```diff
@@ -91,7 +110,12 @@ bool test_main(void) {
     // exit successfully.
     CHECK(curr_state == kDifLcCtrlStateDev, "State transition failed!");
     check_lc_state_transition_count(LcStateTransitionCount + 1);
-    return true;
+    if (kTestIdx == (kTestNum - 1)) {
+      return true;
+    } else {
+      LOG_INFO("Finish LC transtition check and waiting for reboot.");
+      wait_for_interrupt();
```

Comment:
Please add a TODO here - we should get the SW to reset the full chip by itself rather than have to issue POR externally. 

@tjaychen, @moidx, @cfrantz 
Would it be acceptable to request for a CSR bit to reset the full chip instantly, rather than have to go through the whole low power entry / exit (which involves a bunch of IPs)? I dont know if SW will have a usecase for it - but for testing it will definitely be useful.

Reply:
Thanks for filing an issue, yeah a SW reset would be super useful here.